grammars/jsdoc.cson

```diff
@@ -411,6 +413,92 @@
         ]
       }
     ]
+  # Default @param value, highlighted as JavaScript
+  'default-value':
+    'patterns': [
+      # Strings
+      {
+        'begin': '("|\')'
+        'beginCaptures':
+          '0':
+            'name': 'punctuation.definition.string.begin.js'
+        'end': '\\1|(?=\\*/)'
+        'endCaptures':
+          '0':
+            'name': 'punctuation.definition.string.end.js'
+        'name': 'string.quoted.js'
```

Comment:
I noticed that in bd688cc0c02d4e8ae094f47b6ce4b0ad692f6cc5 you simplified the string rules, but shouldn't name still differentiate between single and double quoted strings?